website/client/components/tasks/column.vue

```diff
@@ -394,6 +400,14 @@ export default {
       }
       this.activeFilters[type] = filter;
 
+      if (['scheduled', 'due'].indexOf(filter.label) === -1) {
```

Comment:
Does this stop the Dailies sort order being changed by the user when they're using the "Due" view? If so, that's not the ideal behaviour.

Reply:
Nope. This just prevents an ordering refresh when they click "due"